modules/developer_manual/pages/webdav_api/groups/custom_groups_endpoints.adoc

```diff
@@ -0,0 +1,126 @@
+=== List Groups
+:page-partial:
+:request_method: PROPFIND
+:request_data_file: list-custom-groups.xml
+:request_path_suffix: 
+
+This endpoint returns a list of all custom groups.
+
+include::{partialsdir}/webdav_api/uri_request_table.adoc[]
+
+include::{partialsdir}/webdav_api/core_curl_request.adoc[]
+
+.{request_data_file}
+[source,xml]
+....
+include::example$core/webdav_api/group/request/list-custom-groups.xml[]
+....
+
+Successful requests return two things: 
+
+. An XML payload.
+. A status of `HTTP/1.1 207 Multi-Status`.
+
+You can see an example of the XML payload below.
+The XML payload contains a `response` element for each group.
+
+[source,xml]
+....
+include::example$core/webdav_api/group/response/list-groups-successful-response.xml[]
+....
+
+==== No Results
+
+If there are no custom groups, then a response similar to the following will be returned.
+
+[source,xml]
+....
+include::example$core/webdav_api/group/response/list-groups-no-results-response.xml[]
+....
+
+=== Rename Custom Group
+:request_method: PROPPATCH
+:request_data_file: rename-custom-group.xml
+:request_path_suffix: <$groupId>
+
+This endpoint allows a custom group to be renamed.
+
+NOTE: Only group admins can rename a group.
```

Comment:
they can rename the groups they manage, not sure if it's clear in this sentence

Reply:
Just updated that sentence. I hope it's sufficiently clear now.